web/src/App.tsx

```diff
@@ -12,6 +12,10 @@ import ComparePlayers from './views/ComparePlayers';
 import SalaryPage from './views/SalaryPage';
 import StatsTable from './views/StatsTable';
 import TeamDashboard from './views/TeamDashboard';
+import StatKeeper from './views/StatKeeper';
+import LocalGame from './views/StatKeeper/LocalGame';
+import NewGame from './views/StatKeeper/NewGame'; // Renamed import from NewGameSetup
+import EditGame from './views/StatKeeper/EditGame'; // Renamed import from EditGameRosters
```

Comment:
these comments should be dropped.

Also I wonder if I can sub-router this for more isolation. I do want the StatsKeeper to be self contained, like and app in an app.